### ISSUE TITLE
Possible bug after remote select

### ISSUE THREAD

**Issue by alejoguitar** [gokcehan/lf#1436]
EDIT: Just checked and the problem is on my preview script. I don't have idea why this is happening, as I've been using the same script for years now, just changed the image preview when I switched to wayland. Sorry for the unnecesary report
---
EDIT 2: I sort of answered my own question. With new kitty's behavior (since 0.27, I think) , both image rendering and cleaning needed updating. I only updated the drawing, and it turns out the cleaner function was doing some strange stuff.

I'm not at all sure if this is somehow related with my lf config, zsh config or what. When I use the lf -remote "send $id select <some_file> command, if I open a shell (using the w hotkey), I get this message from zsh: 
zsh: error on TTY read: input/output error
If I try just to enter "exit" to return to lf, zsh crashes and terminal window closes. I tried this in kitty terminal and gnome terminal, same thing.
I tried also modifying the mapping that calls the -remote command. My original mapping was
map <f-3> $lf -remote "send $id select '"$(find . -type f)"'"
but I changed it to eliminate the fzf command and the shell expansion as to:
map <f-3> $lf -remote "send $id select <some_file>"

Same behavior happens with the fzf integration command that I copied from the wiki

This ONLY happens when I press w inmediately after using the -remote command. If after that I just change directory, and then press w, no problem at all. I guess this is why I never noticed it.

**Comment by gokcehan** [comment]
@alejoguitar From what I understand, you already solved your issue so closing this one.

**Comment by alejoguitar** [comment]
Thanks. I sort of did by trying stuff on the preview script. I'm not at all sure I've solved on my end, but quite sure is not an lf problem. Problem seems very related to #1189 as in some configurations of my preview script I was getting Ga=d,d=a\ on the preview window.
I guess is related to the kitty icat stuff.

**Comment by alejoguitar** [comment]
@gokcehan I'm sorry for bringing this back again. Now I'm sure the problem is on the cleaner script, when called from lf.
I just downloaded lf's latest release and I'm using the cleaner script from the wiki
https://github.com/gokcehan/lf/wiki/Previews#with-kitty-and-pistol

I don't get what's the problem with my config. I'm using lf on a PopOs 22.04 on wayland, terminal is Kitty.
If I call the cleaner script manually, there's no problem. Problem is when cleanup if done by lf. 
Commenting out the line setting the previewer definitely solved the issue, but now there's times in which images are stucked on the preview pane (not so much as I expected, luckilly)
It's usable and I binded a key to call the cleaner script manually, but understanding what is the problem is quite beyond my scope. If somebody can give me a hint I would definitely appreciate it.
